packages/components/src/dropdown-menu/stories/index.story.tsx

```diff
@@ -43,39 +43,42 @@ const meta: Meta< typeof DropdownMenu > = {
 };
 export default meta;
 
-const Template: StoryFn< typeof DropdownMenu > = ( props ) => (
-	<div style={ { height: 150 } }>
-		<DropdownMenu { ...props } />
-	</div>
-);
-
-export const Default = Template.bind( {} );
-Default.args = {
-	label: 'Select a direction.',
-	icon: menu,
-	controls: [
-		{
-			title: 'First Menu Item Label',
-			icon: arrowUp,
-			// eslint-disable-next-line no-console
-			onClick: () => console.log( 'up!' ),
-		},
-		{
-			title: 'Second Menu Item Label',
-			icon: arrowDown,
-			// eslint-disable-next-line no-console
-			onClick: () => console.log( 'down!' ),
-		},
+export const Default: StoryObj< typeof DropdownMenu > = {
+	decorators: [
+		( Story ) => (
+			<div style={ { height: 150 } }>
```

Comment:
I noticed these explicit heights are no longer necessary due to #53889, because popovers are now rendering at the end of the document rather than within the each story div, which used to cut them off visually. I'll do a separate cleanup PR.

Reply:
Cleanup at #63480